beacon-chain/forkchoice/nodetree/types.go

```diff
@@ -0,0 +1,53 @@
+package nodetree
```

Comment:
I recommend we find a better name than `nodetree`. It's too generic. Some options:
`protoarrayV1`
`double-linked-tree`